pkg/secrets/vault.go

```diff
@@ -75,6 +75,12 @@ func (c *vaultClient) setItemAtPath(path, field string, content string) error {
 	return c.upstream.UpsertKV(path, data)
 }
 
+func (c *vaultClient) HasItem(itemName string) bool {
+	path := c.pathFor(itemName)
+	_, err := c.upstream.GetKV(path)
+	return vaultclient.IsNotFound(err)
```

Comment:
THis needs to be negated, if it is a notfound, you need to return false, not true

Reply:
:man_facepalming: